tests/SentryLink.test.ts

```diff
@@ -105,13 +105,13 @@ describe('SentryLink', () => {
               report.breadcrumbs as Array<GraphQLBreadcrumb>;
 
             expect(success.category).toBe('graphql.query');
-            expect(success.level).toBe(Severity.Info);
+            expect(success.level).toBe('info' as SeverityLevel);
```

Comment:
Again, the type assertion `as` does not accomplish what you think it does. Consider this:

```typescript
type Foo = 'foo';

const f: Foo = 'bar';
const b = 'bar' as Foo;
```

Only the first assignment will be recognized by TypeScript as an error. `as` is unsafe - it is an assertion and basically tells TypeScript "i know better".